### ISSUE TITLE
@e0eee That seems like you're trying to use `next-i18next` in NextJs middleware, and would be a separate issue.

### ISSUE THREAD

**Issue by e0eee** [isaachinman/next-i18next#1519]
@e0eee That seems like you're trying to use `next-i18next` in NextJs middleware, and would be a separate issue.

`Module not found: Can't resolve 'fs' in '/nextjs/node_modules/next-i18next/dist/commonjs'
Native Node.js APIs are not supported in the Edge Runtime with 'concurrentFeatures' enabled. Found 'fs' imported.
`

_Originally posted by @isaachinman in https://github.com/isaachinman/next-i18next/issues/1518#issuecomment-967113864_

**Comment by isaachinman** [comment]
@e0eee Can you provide a reproducible repo?

**Comment by e0eee** [comment]
@isaachinman https://github.com/e0eee/nextlocale

**Comment by isaachinman** [comment]
@e0eee That's not a minimum repro. Can you please simplify further, or at least elaborate on how to reproduce the issue and what seems to be causing the problem?

**Comment by e0eee** [comment]
`npm run start`
`pages/Index.tsx`
`getStaticProps`
`props: {...(await serverSideTranslations(locale, ['common', 'footer']))}`

**Comment by isaachinman** [comment]
The error warning _very clearly_ points you to the source of the problem, which is the [concurrentFeatures](https://github.com/e0eee/nextlocale/blob/main/next.config.js#L8) experimental feature.

**Comment by isaachinman** [comment]
@e0eee Once any new stable version of React and/or Next land, yes, I always update `next-i18next`.
